src/babel.js

```diff
@@ -288,16 +289,20 @@ export default function({ types: t }) {
     visitor: {
       Program: {
         enter(path, state) {
+          setStateOptions(state)
           state.hasJSXStyle = null
           state.ignoreClosing = null
           state.file.hasJSXStyle = false
           state.file.hasCssResolve = false
-          setStateOptions(state)
-
-          // `addDefault` will generate unique id for the scope
-          state.styleComponentImportName = createReactComponentImportDeclaration(
-            state
-          )
+          // create unique identifier for _JSXStyle component
+          state.styleComponentImportName = path.scope.generateUidIdentifier(
+            STYLE_COMPONENT
+          ).name
+          const importDeclaration = createReactComponentImportDeclaration(state)
+          state.styleImportPath = path.unshiftContainer(
+            'body',
+            importDeclaration
+          )[0]
```

Comment:
Now we can move this to `Program.exit` and avoid to insert it here to then remove it at the end if it was not needed

Reply:
I did this because if it's not been used it could be removed at exit after other transform.

e.g.

```
inject imports
transform to cjs by other plugin
detect usage (not used)
remove tranfromed imports (cjs require)
```

If we directly inject at exit, commonjs plugin doesn't take effect

Reply:
Updated. I thought it won't work with cjs plugin, turns out it could.